src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,32 @@
+.stars {
+  display: flex;
+  margin-top: 17px;
+
+  &__reviews {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin-left: 16px;
+    text-align: right;
+    display: flex;
+    align-items: flex-end;
+    color: $main-text-color;
+  }
+}
+
+.star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+
+  &:last-child {
+    margin-right: 0;
+  }
+}
+
+.stars--4 .star:nth-child(-n + 4) {
```

Comment:
add code for all possible modificators (from 0 to 5)

Reply:
but I need to make 4 stars active, I don’t quite understand